### DISCUSSION TITLE
Longhorn 1.4 upgrade

### DISCUSSION THREAD

**Discussion by Jimmy-SafeCash** [techno-tim/k3s-ansible#206] [Questions] [answered]
Hi @timothystewart6 ,

Firstly, thanks for creating this, it's excellent. I have learnt so much, and have a ton yet to go...

I installed longhorn using the app store in Rancher. It comes with longhorn 1.3.2. Is there any easy way to upgrade to 1.4 which supports trimming volumes (some of mine have become a little unwieldly in size and this would solve it). 

Thanks in advance.

**Comment by timothystewart6** [comment]
Hey!  Sure!  This should be easy.  Just go into the apps section, choose installed apps, then choose Longhorn, then from the menu choose  Edit/Upgrade.  1.4 should be there.  Save and it should upgrade it for it!

**Reply by Jimmy-SafeCash** [reply]
Thanks, unforuntately cannot do this as it's not available in marketplace.

I had to uninstall, and reinstall via helm. It's probably the 'correct' way to install it for future releases.

**Reply by timothystewart6** [reply, marked as answer]
yeah, I always go the helm route because it works with and without rancher!
